src/main/scala/org/broadinstitute/dsde/firecloud/service/StatusService.scala

```diff
@@ -1,63 +1,45 @@
 package org.broadinstitute.dsde.firecloud.service
 
-import akka.actor.{Actor, Props}
+import akka.actor.{Actor, ActorRef, Props}
 import akka.pattern._
-import org.broadinstitute.dsde.firecloud.dataaccess.ReportsSubsystemStatus
-import org.broadinstitute.dsde.firecloud.model.ModelJsonProtocol.impSystemStatus
-import org.broadinstitute.dsde.firecloud.model.{SubsystemStatus, SystemStatus}
+import akka.util.Timeout
 import org.broadinstitute.dsde.firecloud.service.PerRequest.{PerRequestMessage, RequestComplete}
 import org.broadinstitute.dsde.firecloud.service.StatusService.CollectStatusInfo
-import org.broadinstitute.dsde.firecloud.{Application, FireCloudException, FireCloudExceptionWithErrorReport}
+import org.broadinstitute.dsde.workbench.util.health.HealthMonitor.GetCurrentStatus
+import org.broadinstitute.dsde.workbench.util.health.StatusCheckResponse
+import org.broadinstitute.dsde.workbench.util.health.StatusJsonSupport.StatusCheckResponseFormat
 import spray.http.StatusCodes
 import spray.httpx.SprayJsonSupport
 
 import scala.concurrent.{ExecutionContext, Future}
+import scala.concurrent.duration._
 
 /**
  * Created by anichols on 4/5/17.
  */
 object StatusService {
   def props(statusServiceConstructor: () => StatusService): Props = Props(statusServiceConstructor())
 
-  def constructor(app: Application)()(implicit executionContext: ExecutionContext): StatusService = {
-    new StatusService(app)
+  def constructor(healthMonitor: ActorRef)()(implicit executionContext: ExecutionContext): StatusService = {
+    new StatusService(healthMonitor)
   }
 
   case class CollectStatusInfo()
 }
 
-class StatusService (val app: Application)
+class StatusService (val healthMonitor: ActorRef)
                     (implicit protected val executionContext: ExecutionContext) extends Actor with SprayJsonSupport {
+  implicit val timeout = Timeout(1.minute)
```

Comment:
Is this a timeout on getting a response from the `healthMonitor`, or does it apply to how long the health monitor waits for HTTP requests? Would be a nice comment to have.